internal/cypress/config.go

```diff
@@ -53,6 +53,7 @@ type Suite struct {
 	Config           SuiteConfig `yaml:"config,omitempty" json:"config"`
 	ScreenResolution string      `yaml:"screenResolution,omitempty" json:"screenResolution"`
 	Mode             string      `yaml:"mode,omitempty" json:"-"`
+	Timeout          int         `yaml:"timeout,omitempty" json:"timeout"`
```

Comment:
Hm, btw. the `globalTimeout` flag supports `time.Duration`, so we could do it with this one as well to keep things consistent. Food for thought!

Then I could specify the timeout like this:
```yaml
defaults:
  timeout: 10m
```